### PR TITLE
Feed pagination

### PR DESCRIPTION
**Pull request by ramdog**
[#19184365221175]
[#35244649871685]

This change completely removes sorting and filtering for now - need to reimplement.  Ideally, sorting, filtering, and pagination will dynamically add state params to the state without reloading the state, but it seems that's not so easy unfortunately: http://stackoverflow.com/questions/20884551/set-url-query-parameters-without-state-change-using-angular-ui-router

### CODE REVIEW

File: server/api/surf/surf.controller.js [hunk 1]
```diff
@@ -21,48 +21,71 @@ var queryFields = 'user_id '+
...
-    .populate('user_id', 'name boards surfSpots')
-    .populate('friends', 'name')
-    .sort({sessionDate: -1})
-    .lean()
-    .exec(function (err, surfs) {
-      if (err) {
-        return handleError(res, err);
-      }
+  var page = req.query.page || 1;
+  console.log(page);
```
Comment (guilload) [inline server/api/surf/surf.controller.js]: I guess you left this console.log unintentionally :)

Comment (ramdog) [inline server/api/surf/surf.controller.js]: Feedback! :heart_eyes_cat: 

Good catch, will fix.

**Comment by ramdog** [comment]
One other thing: maybe reloading the state on changing of state params isn't so bad... without reloading the state, if the user clicks the back button in the browser, they would probably expect to go to the last change they made (i.e. the last paginated page, or remove the last filter they applied...).

**Comment by ramdog** [comment]
Add this to my sublime config (FYI from @guilload):
`"ensure_newline_at_eof_on_save": true,`

**Pull request merged**